File: app/src/main/java/com/akilimo/mobile/utils/CurrencyCode.java
package com.akilimo.mobile.utils;

import android.util.Log;


import com.mynameismidori.currencypicker.ExtendedCurrency;

import io.sentry.Sentry;

public class CurrencyCode {

    private static String LOG_TAG = CurrencyCode.class.getSimpleName();

    public static ExtendedCurrency getCurrencySymbol(String currencyCode) {
        ExtendedCurrency currency = ExtendedCurrency.getCurrencyByISO(currencyCode);
        try {
            return currency;
        } catch (Exception ex) {
            Sentry.captureException(ex);
        }
        return null;
    }
}

File: app/src/main/java/com/akilimo/mobile/utils/CurrencyCode.kt
package com.akilimo.mobile.utils

import com.mynameismidori.currencypicker.ExtendedCurrency
import io.sentry.Sentry

object CurrencyCode {

    fun getCurrencySymbol(currencyCode: String?): ExtendedCurrency? {
        val currency = ExtendedCurrency.getCurrencyByISO(currencyCode)
        try {
            return currency
        } catch (ex: Exception) {
            Sentry.captureException(ex)
        }
        return null
    }
}
